### DISCUSSION TITLE
How to get the same styling

### DISCUSSION THREAD

**Discussion by csalgo70** [markdoc/markdoc#46] [Q&A] [answered]
I followed the instructions https://markdoc.io/docs/examples/react to create a react site with an express / node js server.

I was able to get the markdown rendered but it doesn't have any of the styles that is rendered by https://markdoc.io/sandbox?mode=preview . 

So my question is how do I get the same styling ?

**Comment by csalgo70** [comment, marked as answer]
Okay figured that in order to get the full styling I need to clone / fork the repo git clone https://github.com/markdoc/docs.git and make changes or add to markdown files under docs folder. You can modify the Components-->Shell-->SideNav.Js too.

**Reply by mfix-stripe** [reply]
Yup! Glad you figured it out @csalgo70 🙏
